### PR TITLE
Fixing user uploaded profiles

### PR DESCRIPTION
**Pull request by kaskranenburgQ**
This PR fixes nr. 2 of the issues in [#issue 4260](https://github.com/quintel/etmodel/issues/4260) on ETModel.
Original issue:
2. Modify profiles page under Flexibility does not function:
See [this link](https://beta.energytransitionmodel.com/scenario/flexibility/curve_upload/upload-curves)

### CODE REVIEW

File: gqueries/output_elements/output_series/user_uploaded_profiles/agriculture_heat_curve.gql [hunk 1]
```diff
@@ -22,7 +22,7 @@
         V(
           FILTER(
           ALL(),
-          "hydrogen && (hydrogen.profile == :'weather/agriculture_heating')"
+          "hydrogen && (hydrogen.group == :'weather/agriculture_heating')"
```
Comment (mabijkerk) [inline gqueries/output_elements/output_series/user_uploaded_profiles/agriculture_heat_curve.gql:25]: This does not work. The hydrogen burner node in agriculture has no `weather/agriculture_heating` group, but `unmet-demand: useable_heat_from_electricity_curve`. Is there a more generic way to query this? For example something like the following may be more robust:

```
FILTER(
  SECTOR(agriculture),
   "hydrogen && useable_heat_output_conversion > 0.0"
)
```

File: gqueries/output_elements/output_series/user_uploaded_profiles/hydrogen_export_curve.gql [hunk 2]
```diff
@@ -3,7 +3,7 @@
       V(
         FILTER(
         ALL(),
-        "hydrogen && (hydrogen.profile == :hydrogen_export)"
+        "hydrogen && (hydrogen.group == :hydrogen_export)"
```
Comment (mabijkerk) [inline gqueries/output_elements/output_series/user_uploaded_profiles/hydrogen_export_curve.gql:6]: This does not match the group on the `energy_export_hydrogen_baseload` node. However, in this case we should update the group on the node to `hydrogen_export`, nice catch!

Comment (mabijkerk) [inline gqueries/output_elements/output_series/user_uploaded_profiles/hydrogen_export_curve.gql:6]: This does not match the curve on the `energy_export_hydrogen_baseload`, though in this case we should change the node itself. Good catch!

File: gqueries/output_elements/output_series/user_uploaded_profiles/hydrogen_import_curve.gql [hunk 3]
```diff
@@ -3,7 +3,7 @@
       V(
         FILTER(
         ALL(),
-        "hydrogen && (hydrogen.profile == :hydrogen_import)"
+        "hydrogen && (hydrogen.group == :hydrogen_import)"
```
Comment (mabijkerk) [inline gqueries/output_elements/output_series/user_uploaded_profiles/hydrogen_import_curve.gql:6]: The same goes for import here.

File: gqueries/output_elements/output_series/user_uploaded_profiles/industry_chemicals_heat_curve.gql [hunk 4]
```diff
@@ -49,7 +49,7 @@
           V(
             FILTER(
             ALL(),
-            "hydrogen && (hydrogen.profile == :industry_chemicals_heat)"
+            "hydrogen && (hydrogen.group == :industry_chemicals_heat)"
```
Comment (mabijkerk) [inline gqueries/output_elements/output_series/user_uploaded_profiles/industry_chemicals_heat_curve.gql:52]: This is similar to the issue with heat in agriculture and should be addressed in a similar way.

File: gqueries/output_elements/output_series/user_uploaded_profiles/industry_fertilizers_heat_curve.gql [hunk 5]
```diff
@@ -49,7 +49,7 @@
           V(
             FILTER(
             ALL(),
-            "hydrogen && (hydrogen.profile == :industry_fertilizers_heat)"
+            "hydrogen && (hydrogen.group == :industry_fertilizers_heat)"
```
Comment (mabijkerk) [inline gqueries/output_elements/output_series/user_uploaded_profiles/industry_fertilizers_heat_curve.gql:52]: This is similar to the issue with heat in agriculture and should be addressed in a similar way.

File: gqueries/output_elements/output_series/user_uploaded_profiles/industry_refineries_heat_curve.gql [hunk 6]
```diff
@@ -49,7 +49,7 @@
           V(
             FILTER(
             ALL(),
-            "hydrogen && (hydrogen.profile == :industry_refineries_heat)"
+            "hydrogen && (hydrogen.group == :industry_refineries_heat)"
```
Comment (mabijkerk) [inline gqueries/output_elements/output_series/user_uploaded_profiles/industry_refineries_heat_curve.gql:52]: This is similar to the issue with heat in agriculture and should be addressed in a similar way.

**Comment by kaskranenburgQ** [comment]
During this PR some underlying issues where found, these are described here:

- #3048 

- #3049 

Our approach is to 'fix' the profiles chart with quick-fixes for now since the deploy is around the corner.

**Review by mabijkerk — APPROVED** [review]

**Pull request merged**